### DISCUSSION TITLE
New transmitter wont connect

### DISCUSSION THREAD

**Discussion by Rinholl** [NightscoutFoundation/xDrip#2667] [Q&A]
Hi, I have been using xdrip for about 2 now
 to follow my sons bg. 3 days ago we changed his sensor and transmitter. Now it won't connect??? This hasn't happened before, I can figure out what is wrong.
![inbound6898191611192584454](https://user-images.githubusercontent.com/124943261/217968685-9d2e96b9-df9f-4b45-8cf8-3b1c52dc583c.jpg)

**Comment by Rinholl** [comment]
I did ask this question on the xdrip Facebook group and was advised to post on here. 

Facebook name Corinne Gibson

**Reply by Navid200** [reply]
Thanks.  Please enable debugging as explained here: https://github.com/NightscoutFoundation/xDrip/wiki/Enabling-Debugging-for-OB1-collector
Then, please go to logs and upload logs.

**Reply by Navid200** [reply]
If you have done that already, please tell me what brand and model is the phone and what Android level is on the phone.

**Reply by Rinholl** [reply]
It is a Samsung s20 
![inbound6315179699054803838](https://user-images.githubusercontent.com/124943261/218188944-6f7b4557-aa3e-4074-972a-dff5f0e29027.jpg)

**Comment by Rinholl** [comment]
![Uploading inbound7229072848321014637.jpg…]()

**Comment by Navid200** [comment]
What is the phone brand? 
What is the model?

**Reply by Rinholl** [reply]
Samsung s20

![Uploading inbound1269890391692298192.jpg…]()

**Reply by Navid200** [reply]
Please post a screenshot of the following page:
Settings &#8722;> G5/G6/Dex1 Debug Settings

**Reply by Rinholl** [reply]
![Screenshot_20230211-094829_xDrip+](https://user-images.githubusercontent.com/124943261/218251655-2e423053-63f6-42ae-8abc-d3fd0b6d4d90.jpg)
![Screenshot_20230211-094848_xDrip+](https://user-images.githubusercontent.com/124943261/218251657-d7e0d7ca-48f9-4df9-b43b-86b91302ec09.jpg)
![Screenshot_20230211-094856_xDrip+](https://user-images.githubusercontent.com/124943261/218251658-5dbcb7ab-d92f-4b2a-a0d9-c405c54bae07.jpg)

**Comment by Navid200** [comment]
What does system status page show now?

**Reply by Rinholl** [reply]
![inbound7326918061343782296](https://user-images.githubusercontent.com/124943261/218263149-7e6b5f2b-db45-42b6-ad82-d6613553ecd2.jpg)

**Reply by Navid200** [reply]
But, that's a different transmitter.
I can help you with that.  But, that defeats the whole purpose of posting here so that the lead developer can see.

What do you want to do?
Why did you switch the transmitter?

**Reply by Navid200** [reply]
You can start sensor.
If after a few minutes, xDrip tells you that sensor start failed, you will need to follow instructions here to perform a restart:
https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

If the sensor has been on the body more than an hour or so, after the warm-up period when the readings start to show, your readings could be in error.  You will need to calibrate.  Please carefully follow the calibration instructions on that same page in order to avoid a confused calibration error.

**Reply by Rinholl** [reply]
I didn't switch it he has the same one on, I don't know why it has the old transmitter ID. I will change the transmitter ID and try to start it again

**Reply by Navid200** [reply]
No, no, no
Please don't do that

**Reply by Rinholl** [reply]
OK I haven't.

**Reply by Navid200** [reply]
If you want to use the new transmitter, please change the transmitter ID to the new one.
Wait 10 minutes.  Then, please post a screenshot of the system status page.

**Reply by Rinholl** [reply]
That has worked. 

Thank you so much. Maybe when I put the ID in it didn't save for some reason.

**Reply by Rinholl** [reply]
![Uploading Screenshot_20230211-150048_xDrip+.jpg…]()
